src/cache/precipitation_precomputed_quantities.jl

```diff
@@ -357,6 +367,11 @@ function set_precipitation_cache!(Y, p, ::Microphysics2Moment, _)
     (; ᶜSqₗᵖ, ᶜSqᵢᵖ, ᶜSqᵣᵖ, ᶜSqₛᵖ) = p.precomputed
     (; ᶜSnₗᵖ, ᶜSnᵣᵖ) = p.precomputed
 
+    q_liq = ᶜspecific(Y.c.ρq_liq, Y.c.ρ)
+    q_rai = ᶜspecific(Y.c.ρq_rai, Y.c.ρ)
+    n_liq = ᶜspecific(Y.c.ρn_liq, Y.c.ρ)
+    n_rai = ᶜspecific(Y.c.ρn_rai, Y.c.ρ)
+
```

Comment:
Same as above: Do we need `q_liq` etc. here, given that they are re-computed in the microphysics functions calls below anyway.